config/data.go

```diff
@@ -0,0 +1,137 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/LANFest/Discord-Bot-Creation/utils"
+	"github.com/ahmetb/go-linq"
+	"github.com/bwmarrin/discordgo"
+)
+
+// Constants : Singleton Instance of ConstantsModel
+func Constants() ConstantsModel {
+	file, err := os.Stat("debug.txt")
```

Comment:
FWIW go does have logging (syslog, etc) and tracing support via standard packages.. which would be preferable here.
Okay for now..

Reply:
https://golang.org/pkg/log/
https://golang.org/pkg/runtime/trace/
there are also libraries like:
https://godoc.org/github.com/golang/glog
and
https://github.com/sirupsen/logrus

Reply:
Tech-286 created to track this future work.